ONSdigital/dp-frontend-router
# Feature/splash page

### What

Add splash page handling to the router

### How to review

* Start the router with `SPLASH_PAGE=dd/splash make debug`
* Have `dp-frontend-renderer` started on `feature/add-dd-splash` branch
* Open http://localhost:20000 (or any URL below that, e.g. http://localhost:20000/dd/datasets)
* Should see the splash page
* Click the button, should see the page you wanted
* Close browser tab (window?) - cookie is session based
* Open page, should see splash page again

### Who can review

Anyone except @ian-kent
